azerothcore/azerothcore-wotlk
# fix(Core/Scripts): Convert Galen's Escape to SAI

## Changes Proposed:
- Partially port TrinityCore/TrinityCore@f621173
- Remove zone_swamp_of_sorrows from Cmake config

## Issues Addressed:

- Closes #5630
- Closes https://github.com/chromiecraft/chromiecraft/issues/522

## SOURCE:

https://www.youtube.com/watch?v=57_CgmmO-hI
https://www.youtube.com/watch?v=Uxvm_uQvQhQ

## Tests Performed:

- Tested in-game

![image](https://user-images.githubusercontent.com/61223313/117427914-c3750480-aee2-11eb-8c04-7934647ed4db.png)

## How to Test the Changes:


.go c id 5391
.q a 1393
Start quest and check based on the source provided if the changes are aligned to the blizzlike behavior of the quest


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
